M/MPICH/build_tarballs.jl

```diff
@@ -59,28 +77,17 @@ if [[ "${target}" == i686-linux-musl ]]; then
     # Small hack: edit `configure` script to force `cross_compiling` to be
     # always "yes".
     sed -i 's/cross_compiling=no/cross_compiling=yes/g' configure
-    EXTRA_FLAGS+=(ac_cv_sizeof_bool="1")
+    configure_flags+=(ac_cv_sizeof_bool="1")
 fi
 
 if [[ "${target}" == aarch64-apple-* ]]; then
-    EXTRA_FLAGS+=(
+    configure_flags+=(
         FFLAGS=-fallow-argument-mismatch
         FCFLAGS=-fallow-argument-mismatch
     )
 fi
 
-# Do not install doc and man files which contain files which clashing names on
```

Comment:
Why was this removed?

Reply:
It wasn't, it (the comment and the option) was just moved.